packages/ruby/lib/readme/har/serializer.rb

```diff
@@ -17,7 +19,7 @@ def initialize(request, response, start_time, end_time, filter)
         @filter = filter
       end
 
-      def to_json
+      def to_json(*_args)
```

Comment:
Though these `to_json` methods don't accept any arguments, it seems to be preferable for them to at least accept arguments, even if we don't do anything with them (which we don't).

https://docs.rubocop.org/rubocop/cops_lint.html#linttojson

Reply:
I have run into this issue before, and had to add back unused args to my `#to_json` methods.